src/adagrams.js

```diff
@@ -1,8 +1,170 @@
 const Adagrams = {
+  letterDistribution: {
+    A : 9,
+    B : 2,	
+    C : 2,	
+    D : 4,	
+    E : 12,	
+    F : 2,	
+    G : 3,	
+    H : 2,	
+    I : 9,	
+    J : 1,	
+    K : 1,	
+    L : 4,	
+    M : 2,
+    N : 6,
+    O : 8,
+    P : 2,
+    Q : 1,
+    R : 6,
+    S : 4,
+    T : 6,
+    U : 4,
+    V : 2,
+    W : 2,
+    X : 1,
+    Y : 2,
+    Z : 1,
+  },
+
+  scoreChart: {
+    A: 1, E: 1, I: 1, O: 1, U: 1, L: 1, N: 1, R: 1, S: 1, T: 1,
+    D: 2, G: 2,
+    B: 3, C: 3, M: 3, P: 3,
+    F: 4, H: 4, V: 4, W: 4, Y: 4,
+    K: 5,
+    J: 8, X: 8,
+    Q: 10, Z:	10,
+  },
+
+  letterPool: [],
+
+  randomize() {
+    const letterPool = [];
+    for (const letter in this.letterDistribution) {
+      for (let counter = 0; counter < this.letterDistribution[letter]; counter++) {
+        this.letterPool.push(letter);
+      }
+    }
+    const randomNumber = Math.floor(Math.random() * this.letterPool.length);
+    const letter = this.letterPool[randomNumber];
```

Comment:
So this is ALMOST perfect! You pull the letter from the pool using the right randomness on the first draw, but instead of *taking* the letter away from the pool, you are just *copying* the letter. So, it's unfortunately possible to end up with (however unlikely it may be) a hand that only has `Q`'s in it! 

Not a big deal, but take some time to look up how to remove an element from an array over break! :)